webapp/packages/core-sdk/src/CustomGraphQLClient.ts

```diff
@@ -30,7 +31,16 @@ export class CustomGraphQLClient extends GraphQLClient {
     );
   }
 
+  blockRequests(): void {
+    this.isRequestsBlocked = true;
+  }
+
   private async overrideRequest<T>(query: string, variables?: Variables): Promise<T> {
+    if(this.isRequestsBlocked) {
+      const e = new Error('Session expired');
+      e.name = 'SessionError';
+      throw e;
```

Comment:
here we should throw blocking reason if it's is `instance of Error`, or `Error` with reason as message